<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_Monster_Speed_Window</class>
 <widget class="QDialog" name="Add_Monster_Speed_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Monster Speeds</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(255, 55, 0, 255);</string>
  </property>
  <widget class="QLabel" name="Main_Menu_Label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Speeds&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Speeds_Walk">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>130</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Walk&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Burrow&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Speeds_Burrow">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Fly&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Speeds_Fly">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Climb&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Speeds_Climb">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Speeds_Swim">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>130</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Swim&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Speeds_Save_Speeds_Button">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>170</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);</string>
   </property>
   <property name="text">
    <string>Save Speeds</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Speeds_Save_Speeds_Button</sender>
   <signal>clicked()</signal>
   <receiver>Add_Monster_Speed_Window</receiver>
   <slot>Add_Monster_Speeds_Save_DB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>1466</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Add_Monster_Speeds_Save_DB()</slot>
 </slots>
</ui>
